examples/transformation/main.cpp

```diff
@@ -195,14 +195,22 @@ static int capture(std::string output_dir, uint8_t deviceId = K4A_DEVICE_DEFAULT
     }
 
     // Compute color point cloud by warping color image into depth camera geometry
+#ifdef _WIN32
```

Comment:
Windows will actually accept forward slashes as valid path separators. We may be able to drop the #if logic here.